components/modal/src/Modal.tsx

```diff
@@ -182,84 +172,81 @@ export const Modal = ({
 		}
 	}, [onClose]);
 
-	// on escape close modal
-	const keyHandler = useCallback(
-		(e: KeyboardEvent) => {
-			if (dismissible && open && e.keyCode === 27) handleClose();
-		},
-		[dismissible, handleClose, open]
-	);
```

Comment:
This code was not working, it was meant to close the modal if it was dismissable and someone hit escape. The react-focus-on library FocusOn component has a prop that enables this functionality, that has been implemented and tested, it works